README.Rmd

```diff
@@ -12,24 +12,30 @@ knitr::opts_chunk$set(
   )
 ```
 
-# evalITR
+
+# evalITR: Estimation and Evaluation Methods for Individualized Treatment Rules
 
 <!-- badges: start -->
+```{r Workflow, echo=FALSE, out.width = '100%'}
+knitr::include_graphics("man/figures/README-manual.png")
+```
 <!-- badges: end -->
 
-## Installation
+This R package enables researchers to make statistical inference with any learning algorithm to estimate and validate individualized treatment rules with experimental data. The package is modularized and integrates with various state-of-the-art machine learning packages. It allows parameter tuning and provides original ITR evaluation metrics as well as hypothesis testing. Users will be able to complete the entire workflow of analysis i.e., training, estimation, and validation in one step. Computation of various summary statistics and plotting functionality are also included. 
 
-You can install the development version of evalITR from [GitHub](https://github.com/) with:
+## Installation Instructions
 
-``` r
+You can install the development version of 'evalITR' from [GitHub](https://github.com/) with:
+
+```{r}
 # install.packages("devtools")
-devtools::install_github("MichaelLLi/evalITR")
+devtools::install_github("xiaolong-y/evalITR@causal-ml")
```

Comment:
Please use Michael's original directory for the documentation.

Reply:
Yes! I did this temporarily for this case because of the new functions included in the read.rmd file were not merged to Michael's repository just yet.